ostd/src/arch/x86/boot/smp.rs

```diff
@@ -37,9 +38,17 @@ use crate::{
             Level, TriggerMode,
         },
     },
+    if_tdx_enabled,
     mm::{paddr_to_vaddr, PAGE_SIZE},
 };
 
+cfg_if! {
+    if #[cfg(feature = "cvm_guest")] {
+        use crate::arch::x86::kernel::acpi::AcpiMemoryHandler;
+        use acpi::platform::wakeup_aps;
```

Comment:
For clarity, this entire `cfg_if` import block should be merged into the code block that actually uses these imports. Because that is the only user of these imports.

```rust
    if_tdx_enabled!({
        use crate::arch::x86::kernel::acpi::AcpiMemoryHandler;
        use acpi::platform::wakeup_aps;

        for ap_num in 1..num_cpus {
            wakeup_aps(
                &ACPI_TABLES.get().unwrap().lock(),
                AcpiMemoryHandler {},
                ap_num,
                AP_BOOT_START_PA as u64,
                1000,
            )
            .unwrap();
        }
    } else {
        send_boot_ipis();
    });
```